lib/auth/server.ts

```diff
@@ -60,77 +82,29 @@ export const awaitForToken = (options: {
 	const { app, server } = createServer({ port: options.port });
 
 	return new Promise<string>((resolve, reject) => {
-		const closeServer = (
-			errorMessage: string | undefined,
-			successPayload?: string,
-		) => {
-			server.close(() => {
-				if (errorMessage) {
-					reject(new Error(errorMessage));
-					return;
-				}
-
-				resolve(successPayload);
-			});
-		};
-
-		const renderAndDone = async ({
-			request,
-			response,
-			viewName,
-			errorMessage,
-			statusCode = 200,
-			token,
-		}: {
-			request: express.Request;
-			response: express.Response;
-			viewName: 'success' | 'error';
-			errorMessage?: string;
-			statusCode?: number;
-			token?: string;
-		}) => {
-			const context = await getContext(viewName);
-			response.status(statusCode).render(viewName, context);
-			request.connection.destroy();
-			closeServer(errorMessage, token);
-		};
-
 		app.post(options.path, async (request, response) => {
+			server.close(); // stop listening for new connections
```

Comment:
Previously, `server.close()` was called at the end of `app.post`, now it is called at the beginning. Shouldn't matter much, as all it does is to stop accepting new connections (which frees up the listening port, and important detail for the existing automated tests). Note that the images and CSS used in the web page served by the CLI are inlined (base64 for the images), so the browser does not need to make additional HTTP requests to the CLI server.